ext/libgecode3/extconf.rb

```diff
@@ -25,9 +29,17 @@ def self.prefix
     PREFIX
   end
 
+  def self.sh
+    if openbsd?
+      "bash"
+    else
+      "sh"
+    end
+  end
```

Comment:
this should probably generically try "bash" if its in the path (with a little help to make sure /usr/local/bin is in the path) and then resolve to the full path of bash, and then fallback to /bin/sh